app/views/admins/dashboard.html.erb

```diff
@@ -0,0 +1,24 @@
+<h1>This is the admin dashboard</h1>
+
+<%= link_to 'merchants', admin_merchants_path %>
+<%= link_to 'invoices', admin_invoices_path %>
+
+
+<h2>Top 5 Customers</h2>
+<section id="top-customers">
+  <ul>
+    <% @customers.each do |customer| %>
+        <li id="customer-<%= customer.id %>"><%= customer.first_name + " " + customer.last_name + " :#{customer.number_of_successful_transactions}" %></li>
```

Comment:
we created a `.name` method for customers that concatenates the first and last name but I think in actuality it would be better off as a method in ApplicationHelper that we could call on in multiple views

Reply:
As an exercise in debating where things go - I don't think this fits in application helper because it only applies to Customer, HOWEVER, it is a view logic, so as you all have done here it is not out of place in the view. That being said it is used a few times, so my vote would be on making a partial just for displaying the name, but @cunninghamge vetoed that... So many ways to do it